### DISCUSSION TITLE
收到 Strava 的邮件: "Apply to Strava’s Developer Program"

### DISCUSSION THREAD

**Discussion by aJIEw** [yihong0618/running_page#554] [General]
> Here’s a friendly reminder to apply to Strava’s new Developer Program by March 4th, 2024. If you don’t, you’ll lose access to our API and miss out on new program features and benefits.

让你提交一个 form，里面包含 client id 以及同意 brand guideline (需要在网页或者应用中添加一个 Connect with Strava 按钮)

这个如果不填的话会影响继续使用开发者接口吗？

**Comment by yihong0618** [comment]
会影响，好像是到明年。。

**Comment by aJIEw** [comment]
再次收到 Strava 的邮件通知，当只有一个人使用的时候不用申请 Developer Program 了：

> Hi there,
>
> We've identified your Strava API application as operating in single player mode as you only have one authenticated athlete.
>
> That means you don't need to take any additional action to formally submit to our Developer Program if you intend to continue developing with just your own data. 
>
> Moving forward, here’s what you can expect for your app:
>
> - **Overall Rate Limit**: allows for 200 requests every 15 min, with up to 2,000 requests per day
> - **Read Rate Limit**: allows for 100 requests every 15 min, with up to 1,000 requests per day
> - **Athlete Capacity**: 1

👏👏

**Reply by yihong0618** [reply]
@aJIEw 我没太懂，这个是好事情是吧

**Reply by aJIEw** [reply]
是的，可以继续用，什么都不用改

**Reply by yihong0618** [reply]
好的感谢
